lib/fishjam_web/controllers/peer_controller.ex

```diff
@@ -86,19 +89,30 @@ defmodule FishjamWeb.PeerController do
       |> render("show.json", assigns)
     else
       :error ->
-        {:error, :bad_request, "Invalid request body structure"}
+        msg = "Invalid request body structure"
+        Logger.warning(msg)
```

Comment:
Shouldn't we specify the action that failed? I.e. `"Unable to add peer to room #{room_id}: " <> msg`?